djangocms_navigation/admin.py

```diff
@@ -362,7 +426,7 @@ def get_queryset(self, request):
         return self.model().get_tree()
 
     def change_view(
-        self, request, object_id, menu_content_id=None, form_url="", extra_context=None
+            self, request, object_id, menu_content_id=None, form_url="", extra_context=None
```

Comment:
```suggestion
        self, request, object_id, menu_content_id=None, form_url="", extra_context=None
```
Some additional formatting added unrelated to the changes being made here.